### ISSUE TITLE
training my custom dataset problem

### ISSUE THREAD

**Issue by Linengyao** [nianticlabs/monodepth2#504]
I've seen your comments on weak or untextured datasets in a lot of issues. I am currently ranging sea surface targets, and do not use the pre-trained model to train from 0, I found that the loss of the model at the beginning is very small (10e-6 level), but the overall loss is still showing a downward trend, please ask if this model is suitable for the use of this scenario, the following is the dataset picture and the loss picture. And I found that the output of your model is between 0.1-100, not the real depth value, if I need to output the real depth value (distance value) what should I do?
![ef44bd50a5387a301c44f42757bc1c5](https://github.com/user-attachments/assets/2d3673b4-81bf-4b8d-8cb2-72d6fa5d4bb4)

<img width="861" alt="a314ad1362a2c978246450d749be4c0" src="https://github.com/user-attachments/assets/34545f4a-5f07-4a08-9dc1-6737211d7efc">
